code/__DEFINES/maps.dm

```diff
@@ -71,6 +71,7 @@ require only minor tweaks.
     ZTRAIT_BOMBCAP_MULTIPLIER = 2, \
     ZTRAIT_BASETURF = /turf/open/lava/smooth/lava_land_surface)
 #define ZTRAITS_REEBE list(ZTRAIT_REEBE = TRUE, ZTRAIT_BOMBCAP_MULTIPLIER = 0.5)
+#define ZTRAITS_AWAY list(ZTRAIT_AWAY = TRUE, ZTRAIT_BOMBCAP_MULTIPLIER = 0)
```

Comment:
Any reason for disabling maxcaps in away missions?

Reply:
So they can't be chees'd.

Reply:
Because you cannot have the current away missions fastmos, explosions and deconstructable terrain at the same time. A single space tile will rapidly decompress most away missions. The only way to allow explosions on away missions with fastmos would be to make every single tile and wall indestructible, as well as disallowing building. Getting rid of explosions is simply the least disruptive solution.

Reply:
Still that doesn't limit chemical grenades